### ISSUE TITLE
Minecraft crash after joining a world [1.19.3-1.20.1]

### ISSUE THREAD

**Issue by Picsald** [mega12345mega/NBT-Editor#76]
Hi my minecraft crash every time I join a server and according to  _Not Enough Crashes_  the issue seems be with NBT Editor.
Here is my Crash report :

```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 2024-01-07 15:46:42
Description: Unexpected error

java.lang.NoClassDefFoundError: net/minecraft/class_8673
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.commands.ClientCommandManager.lambda$static$0(ClientCommandManager.java:117)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.Reflection.lambda$getOptionalMethod$2(Reflection.java:140)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.Reflection$1.get(Reflection.java:153)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.commands.ClientCommandManager.lambda$createDispatcher$3(ClientCommandManager.java:128)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.Version$VersionSwitch.get(Version.java:54)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.commands.ClientCommandManager.createDispatcher(ClientCommandManager.java:134)
	at net.minecraft.class_634.handler$dcj002$nbteditor$onGameJoin(class_634.java:12256)
	at net.minecraft.class_634.method_11120(class_634.java:465)
	at net.minecraft.class_2678.method_11567(class_2678.java:95)
	at net.minecraft.class_2678.method_11054(class_2678.java:25)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:156)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:130)
	at net.minecraft.class_1255.method_5383(class_1255.java:115)
	at net.minecraft.class_310.method_1523(class_310.java:1175)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_8673
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 21 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.commands.ClientCommandManager.lambda$static$0(ClientCommandManager.java:117)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.Reflection.lambda$getOptionalMethod$2(Reflection.java:140)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.Reflection$1.get(Reflection.java:153)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.commands.ClientCommandManager.lambda$createDispatcher$3(ClientCommandManager.java:128)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.Version$VersionSwitch.get(Version.java:54)
	at com.luneruniverse.minecraft.mod.nbteditor.multiversion.commands.ClientCommandManager.createDispatcher(ClientCommandManager.java:134)
	at net.minecraft.class_634.handler$dcj002$nbteditor$onGameJoin(class_634.java:12256)
	at net.minecraft.class_634.method_11120(class_634.java:465)
	at net.minecraft.class_2678.method_11567(class_2678.java:95)
	at net.minecraft.class_2678.method_11054(class_2678.java:25)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:156)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:130)

-- Affected level --
Details:
	All players: 1 total; [class_746['Picsald'/1736245, l='ClientLevel', x=8.50, y=65.00, z=8.50]]
	Chunk stats: 1024, 0
	Level dimension: minecraft:ville_2
	Level spawn location: World: (8,64,8), Section: (at 8,0,8 in 0,4,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 0 game time, 0 day time
	Server brand: ~~NULL~~
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:458)
	at net.minecraft.class_310.method_1587(class_310.java:2406)
	at net.minecraft.class_310.method_1514(class_310.java:826)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric

-- System Details --
Details:
	Minecraft Version: 1.20
	Minecraft Version ID: 1.20
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 17.0.8.1, Azul Systems, Inc.
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Azul Systems, Inc.
	Memory: 567658720 bytes (541 MiB) / 1149239296 bytes (1096 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 PRO 3600 6-Core Processor          
	Identifier: AuthenticAMD Family 23 Model 113 Stepping 0
	Microarchitecture: Zen 2
	Frequency (GHz): 3.59
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce GTX 1050 Ti
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1c82
	Graphics card #0 versionInfo: DriverVersion=31.0.15.3699
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 23475.24
	Virtual memory used (MB): 15844.65
	Swap memory total (MB): 7168.00
	Swap memory used (MB): 109.48
	JVM Flags: 2 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx2048M
	Fabric Mods: 
		advancementinfo: AdvancementInfo 1.20-fabric0.83.0-1.4
		animatica: Animatica 0.6+1.20
		appleskin: AppleSkin 2.5.1+mc1.20
		ash_api: Ash API 2.1.2+1.20
		autoclicker-fabric: Auto Clicker 1.20-1.7.0-build.24
		autoswitch: AutoSwitch 7.0.2
			autoswitch-api: AutoSwitch API 3.0.0
		axolotlbuckets: Axolotl Buckets 1.2.0
		beenfo: Beenfo 1.20-fabric0.83.0-1.3.3
			crowdin-translate: CrowdinTranslate 1.4+1.19.3
			gbfabrictools: GBfabrictools 1.3.5+1.20
		betterf3: BetterF3 7.0.2
		boatiview: Boat Item View Fabric 0.0.5
		carpet: Carpet Mod 1.4.112+v230608
		cateyes: Cat Eyes 7.3.0
		catjammies: Cat Jammies 1.0.0-1.20.1
		chat_heads: Chat Heads 0.10.30
		chesttracker: Chest Tracker 1.1.23
			whereisit: Where Is It 1.14.17
		cloth-config: Cloth Config v11 11.1.118
			cloth-basic-math: cloth-basic-math 0.6.1
		cmdkeybind: Command Macros 1.6.3-1.20
			kyrptconfig: Kyrpt Config 1.5.6-1.20
		collective: Collective 6.57
		continuity: Continuity 3.0.0-beta.2+1.20
		controlling: Controlling For Fabric 12.0.2
		cosmetica: Cosmetica 1.2.7
			com_github_cosmetica-cc_cosmeticadotjava: CosmeticaDotJava 1.11.2
			fabric-api-base: Fabric API Base 0.4.29+b04edc7a27
			sulphate: Sulfate 1.1.1
		craftpresence: CraftPresence 2.2.6
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.11.9+132c48c177
		dashloader: DashLoader 5.0.0-beta.2+1.20.0
			com_github_luben_zstd-jni: zstd-jni 1.5.2-2
			dev_notalpha_taski: Taski 2.1.0
			dev_quantumfusion_hyphen: Hyphen 0.4.0-rc.3
		easiercrafting: EasierCrafting 1.20-fabric0.83.0-1.7
		enchantips: Enchantips 1.2.0
		enchantment-reveal: Enchantment Reveal 1.0.0
		entity_model_features: Entity Model Features 1.2.1
		entity_texture_features: Entity Texture Features 5.1.2
			org_apache_httpcomponents_httpmime: httpmime 4.5.10
		entityculling: EntityCulling-Fabric 1.6.2-mc1.20.1
		extremesoundmuffler: Extreme sound muffler 3.39-fabric-1.20
			fiber: fiber 0.23.0-2
		fabric-api: Fabric API 0.83.0+1.20
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.34+4d8536c927
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.10+b3afc78b27
			fabric-block-api-v1: Fabric Block API (v1) 1.0.9+e022e5d127
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.39+b3afc78b27
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.20+b3afc78b27
			fabric-command-api-v1: Fabric Command API (v1) 1.2.32+f71b366f27
			fabric-command-api-v2: Fabric Command API (v2) 2.2.11+b3afc78b27
			fabric-commands-v0: Fabric Commands (v0) 0.2.49+df3654b327
			fabric-containers-v0: Fabric Containers (v0) 0.1.61+df3654b327
			fabric-content-registries-v0: Fabric Content Registries (v0) 4.0.7+b3afc78b27
			fabric-convention-tags-v1: Fabric Convention Tags 1.5.3+b3afc78b27
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.18+aeb40ebe27
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 12.1.10+b3afc78b27
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.51+b3afc78b27
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.21+b3afc78b27
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.6.0+b3afc78b27
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.61+df3654b327
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.38+b04edc7a27
			fabric-item-api-v1: Fabric Item API (v1) 2.1.26+b3afc78b27
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.0.7+b3afc78b27
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.36+fb8d95da27
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.34+df3654b327
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.20+b3afc78b27
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.37+b3afc78b27
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.41+9e7660c627
			fabric-message-api-v1: Fabric Message API (v1) 5.1.6+b3afc78b27
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.47+b3afc78b27
			fabric-models-v0: Fabric Models (v0) 0.3.35+b3afc78b27
			fabric-networking-api-v1: Fabric Networking API (v1) 1.3.8+b3afc78b27
			fabric-networking-v0: Fabric Networking (v0) 0.3.48+df3654b327
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 11.0.6+b3afc78b27
			fabric-particles-v1: Fabric Particles (v1) 1.0.28+b3afc78b27
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.18+b3afc78b27
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.2.6+b3afc78b27
			fabric-renderer-api-v1: Fabric Renderer API (v1) 3.0.1+b3afc78b27
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.3.1+b3afc78b27
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.44+df3654b327
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.33+b3afc78b27
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.26+b3afc78b27
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.47+df3654b327
			fabric-rendering-v1: Fabric Rendering (v1) 3.0.6+b3afc78b27
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.4+b3afc78b27
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.6+b3afc78b27
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.27+b3afc78b27
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.12+b3afc78b27
			fabric-transfer-api-v1: Fabric Transfer API (v1) 3.2.2+b3afc78b27
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 4.2.0+b3afc78b27
		fabric-language-kotlin: Fabric Language Kotlin 1.10.17+kotlin.1.9.22
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.9.22
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.9.22
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.9.22
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.9.22
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.23.1
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.7.3
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.7.3
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.5.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.6.2
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.6.2
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.6.2
		fabricloader: Fabric Loader 0.15.3
			mixinextras: MixinExtras 0.3.2
		flighthelper: FlightHelper 1.20-fabric0.83.0-1.2.2
		forgeconfigapiport: Forge Config API Port 7.0.0
		freecam: Freecam (Modrinth Edition) 1.2.1+1.20
		freelook: freelook 1.1.3
		horsestatsvanilla: Horse Stats Vanilla 4.3.0
		inventory_control_tweaks: Inventory Control Tweaks 1.3.27
		inventoryhud: Inventory HUD + 3.4.18
		inventoryprofilesnext: Inventory Profiles Next 1.10.9
		java: OpenJDK 64-Bit Server VM 17
		krypton: Krypton 0.2.3
			com_velocitypowered_velocity-native: velocity-native 3.2.0-SNAPSHOT
		lazydfu: LazyDFU 0.1.3
		libipn: libIPN 4.0.1
		litematica: Litematica 0.15.0
		lithium: Lithium 0.11.2
		malilib: MaLiLib 0.16.3
		maptooltip: Map Tooltip 3.0.0
		memoryleakfix: Memory Leak Fix 1.1.5
		minecraft: Minecraft 1.20
		minihud: MiniHUD 0.27.0
		modelfix: Model Gap Fix 1.10
		modmenu: Mod Menu 7.0.1
		morechathistory: MoreChatHistory 1.1.1
		mousewheelie: Mouse Wheelie 1.11.1+mc1.20-pre2
			amecsapi: Amecs API 1.4.0+mc1.20-pre1
			coat: Coat 1.0.0-beta.20+mc1.20-pre1
			tweed4_annotated: tweed4_annotated 1.3.1+mc1.20-pre1
			tweed4_base: tweed4_base 1.7.1+mc1.20-pre1
			tweed4_data: tweed4_data 1.2.1+mc1.20-pre1
			tweed4_data_hjson: tweed4_data_hjson 1.1.1+mc1.20-pre1
			tweed4_tailor_coat: tweed4_tailor_coat 1.1.3+mc1.20-pre1
			tweed4_tailor_lang_json_descriptions: tweed4_tailor_lang_json_descriptions 1.1.0+mc1.20-pre1
			tweed4_tailor_screen: tweed4_tailor_screen 1.1.4+mc1.20-pre1
		name_visibility: Better Name Visibility 1.1
		nbt_ac: NBT Autocomplete 1.1.8
		nbteditor: NBT Editor 1.12.3
		no-hide-flags: No Hide Flags 1.1.0+1.20
		notenoughcrashes: Not Enough Crashes 4.4.4+1.20
		noweathereffects: No Weather Effects 1.7.0
		openpartiesandclaims: Open Parties and Claims 0.20.4
		preventer: Preventer 1.1.0
		reeses-sodium-options: Reese's Sodium Options 1.6.3+mc1.20.1-build.92
		removewardeneffect: No darkness effect 0.0.2-1.20
		replanter: Replanter 1.3
		restorechatlinks: Restore chat links 1.19.3-0.2.1
		searchables: Searchables 1.0.2
		seedcrackerx: SeedCrackerX 2.14.4
			com_seedfinding_latticg: latticg 1.6
			com_seedfinding_mc_biome: mc_biome 41a42cb9019a552598f12089059538853e18ec78
			com_seedfinding_mc_core: mc_core d64d5f90be66300da41ef58f4f1736db2499784f
			com_seedfinding_mc_feature: mc_feature 919b7e513cc1e87e029a9cd703fc4e2dc8686229
			com_seedfinding_mc_math: mc_math ffd2edcfcc0d18147549c88cc7d8ec6cf21b5b91
			com_seedfinding_mc_noise: mc_noise 7e3ba65e181796c4a2a1c8881d840b2254b92962
			com_seedfinding_mc_reversal: mc_reversal ca64c0890c106f1a2207623c316fce86f250b918
			com_seedfinding_mc_seed: mc_seed 1ead6fcefe7e8de4b3d60cd6c4e993f1e8f33409
			com_seedfinding_mc_terrain: mc_terrain b4246cbd5880c4f8745ccb90e1b102bde3448126
		shuffle: Shuffle 8.1.0
		shulkerboxtooltip: Shulker Box Tooltip 4.0.3+1.20
		skinlayers: 3d Skin Layers 1.5.4-mc1.20.1
		sodium: Sodium 0.4.10+build.27
		starlight: Starlight 1.1.2+fabric.dbc156f
		statssearch: StatsSearch 1.20-fabric0.83.0-1.0.1
		stendhal: Stendhal 1.4.1-1.20
			libgui: LibGui 8.0.1+1.20
				jankson: Jankson 5.0.1+j1.2.2
				libninepatch: LibNinePatch 1.2.0
		symbol-chat: Symbol Chat 1.20-1.0.0
		tooltipfix: ToolTip Fix 1.1.1-1.20
		vectorientation: Vectorientation 1.0.1a
		visiblebarriers: Visible Barriers 2.0.9
		voicechat: Simple Voice Chat 1.20-2.4.9
		wakes: Wakes 0.2.2
			blue_endless_jankson: jankson 1.2.2
			com_github_jdiemke_delaunay-triangulator_delaunaytriangulator: DelaunayTriangulator 1.0.0
		xaerominimap: Xaero's Minimap 23.9.3
		xaeroplus: XaeroPlus 1.20.1
		xaeroworldmap: Xaero's World Map 1.37.2
		yet_another_config_lib_v3: YetAnotherConfigLib 3.2.1+1.20
			com_twelvemonkeys_common_common-image: common-image 3.10.0-SNAPSHOT
			com_twelvemonkeys_common_common-io: common-io 3.10.0-SNAPSHOT
			com_twelvemonkeys_common_common-lang: common-lang 3.10.0-SNAPSHOT
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.10.0-SNAPSHOT
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.10.0-SNAPSHOT
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.10.0-SNAPSHOT
			org_quiltmc_parsers_gson: gson 0.2.1
			org_quiltmc_parsers_json: json 0.2.1
	Launched Version: 1.20
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2 GL version 3.2.0 NVIDIA 536.99, NVIDIA Corporation
	Window size: 1920x1032
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: fabric
	Current Language: en_us
	CPU: 12x AMD Ryzen 5 PRO 3600 6-Core Processor 
	Client Crashes Since Restart: 1
	Integrated Server Crashes Since Restart: 0
	Suspected Mods: Minecraft (minecraft), Fabric Loader (fabricloader), NBT Editor (nbteditor)
```

**Comment by mega12345mega** [comment]
After looking into the code, it seems in 1.19.3-1.20.1, joining any world causes the game to crash (not just multiplayer).

**Comment by mega12345mega** [comment]
I've now fixed this in v1.13.0!
